src/Form/EmbedButtonDeleteForm.php

```diff
@@ -30,20 +22,4 @@ public function getCancelUrl() {
     return new Url('embed_button.list');
```

Comment:
If I remove this function also, the file will essentially be empty. In that case, can I directly use the EntityDeleteForm as handler for delete?